Entities/Items/Food/Eatable.as

```diff
@@ -74,7 +74,9 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid)
 		return;
 	}
 
-	if (getNet().isServer() && !blob.hasTag("dead"))
+	if (getNet().isServer() 
+	&& !blob.hasTag("dead")
+	&& !this.hasTag("egg projectile"))
```

Comment:
```suggestion
	if (isServer() && !blob.hasTag("dead"))
```
adding the egg projectile check here is unneeded if other suggestion is applied.